packages/instant/src/containers/latest_buy_quote_order_details.ts

```diff
@@ -0,0 +1,26 @@
+import { BuyQuoteInfo } from '@0xproject/asset-buyer';
+import { BigNumber } from '@0xproject/utils';
+import * as _ from 'lodash';
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import { State } from '../redux/reducer';
+
+import { OrderDetails } from '../components/order_details';
+
+export interface LatestBuyQuoteOrderDetailsProps {}
+
+interface ConnectedState {
+    buyQuoteInfo?: BuyQuoteInfo;
+    ethUsdPrice?: BigNumber;
+}
+
+const mapStateToProps = (state: State, _ownProps: LatestBuyQuoteOrderDetailsProps): ConnectedState => ({
+    // use the worst case quote info
+    buyQuoteInfo: _.get(state, 'latestBuyQuote.worstCaseQuoteInfo'),
```

Comment:
minor: I would prefer to grab the worstCaseInfo without using `_.get` to keep the typesafety (i.e. ensuring `worstCaseQuoteInfo` is actually a property of `latestBuyQuote`

Reply:
I added a package https://github.com/rimeto/ts-optchain. The API is a bit ugly but it gives us the best of both worlds. Open to discussing, but didn't take long to include.